tests/coreos-install/util/util.go

```diff
@@ -106,14 +115,43 @@ func FetchLocalImage(t *testing.T) string {
 	return tmpDir
 }
 
-func DownloadFile(tmpDir, name string) error {
+// Used to get defaults for channel, board, & version, first checks if the
+// host machine is Container Linux and if so uses the data from the machine
+// otherwise defaults to stable, amd64-usr, & current respectively
+func GetDefaults(t *testing.T) (string, string, string) {
```

Comment:
If I read
```go
channel, _, _ := GetDefaults(t)
```
as is in the next commit, it's not obvious what `_` and `_` are, or what they're the defaults of.  Better name?